### PR TITLE
bigint constants refactored from config

### LINKED ISSUE #874: Remove constants from config
There are still several maths constants in the default config file:
- BABYJUBJUB parameters
- ELIGATOR2 parameters
- BN128_GROUP_ORDER
- BN128_PRIME_FIELD
- MAX_PUBLIC_VALUES

These should be moved to the constants file

### PR DESCRIPTION
**Pull request by LijuJoseJJ**
fixes #874

### CODE REVIEW

**Comment by israelboudoux** [comment]
Hey @LijuJoseJJ, could you solve the merge conflicts please?

File: common-files/constants/bigInts.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-extend-native */
+BigInt.prototype.toJSON = function () {
```
Comment (pawelgrzybek) [inline common-files/constants/bigInts.js:2]: Looks like we don't use this `toJSON` at all in the codebase. Good time to get rid of it.

**Review by pawelgrzybek — CHANGES_REQUESTED** [review]

File: common-files/constants/bigInts.js (hunk 1 above)
Comment (pawelgrzybek) [reply common-files/constants/bigInts.js:2]: I already spoke about it with @IlyasRidhuan, this comment is not relevant anymore. Sorry

**Review by RogerTaule — APPROVED** [review]
LGTM

**Review by signorecello — APPROVED** [review]
LGTM

**Pull request merged**